server/mailet/integration-testing/src/test/resources/eml/message-multipart-7bit.eml

```diff
@@ -0,0 +1,142 @@
+Return-Path: <bob@james.org>
+MIME-Version: 1.0
+Delivered-To: bob@james.org
+Received: from 10.233.68.83 (EHLO incoming.james.org) ([10.233.68.83])
+          by james-0 (JAMES SMTP Server ) with ESMTP ID -973333758
+          for <bob@james.james.org>;
+          Wed, 22 Jan 2020 03:47:32 +0000 (UTC)
+Received: from smtp.james.org (unknown [10.233.65.0])
+	by incoming.james.org (Postfix) with ESMTPS id 8151D43
+	for <bob@james.james.org>; Wed, 22 Jan 2020 03:47:32 +0000 (UTC)
+Received: from [10.116.29.102] (unknown [1.54.162.156])
+	(using TLSv1.2 with cipher ECDHE-RSA-AES128-GCM-SHA256 (128/128 bits))
+	(No client certificate requested)
+	by smtp.james.org (Postfix) with ESMTPSA id D6C653F38C
+	for <bob@james.org>; Wed, 22 Jan 2020 04:47:31 +0100 (CET)
+To: bob@james.org
+X-LINAGORA-Copy-Delivery-Done: 1
+From: Uncle Bob <bob@james.org>
+Subject: multipart
+Message-ID: <69bd7da2-5b13-de50-bd77-c985f6b0215c@james.org>
+Date: Wed, 22 Jan 2020 10:47:17 +0700
+User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101
+ Thunderbird/60.2.1
+Content-Type: multipart/mixed;
+ boundary="------------C3FD44EAF9B6A93E163D9176"
+Content-Language: en-US
+
+This is a multi-part message in MIME format.
+--------------C3FD44EAF9B6A93E163D9176
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: quoted-printable
+
+A 8bit encoded body with =E2=82=ACuro symbol.

```

Comment:
it looks really like 7bits
```
$ echo -ne "A 8bit encoded body with =E2=82=ACuro symbol." | hexdump -C
00000000  41 20 38 62 69 74 20 65  6e 63 6f 64 65 64 20 62  |A 8bit encoded b|
00000010  6f 64 79 20 77 69 74 68  20 3d 45 32 3d 38 32 3d  |ody with =E2=82=|
00000020  41 43 75 72 6f 20 73 79  6d 62 6f 6c 2e           |ACuro symbol.|
0000002d
```

Reply:
(you can easily check, all that is < 7F / 127 is 7bit. 8bit is bitween 80 and FF)

Reply:
oops by the way it's written in the filename, so you are aware it's 7bit. Maybe you mean extended ascii encoded body? Or Unicode with non ascii characters body?

Reply:
I meant 7bit encoded body